article_store/articles/models.py

```diff
@@ -0,0 +1,22 @@
+import uuid
+
+from django.db import models
+
+
+PUBLISHED = 'Published'
+UNPUBLISHED = 'Unpublished'
+
+ARTICLE_STATUSES = (
```

Comment:
Might want to think about this being configurable (eg you might want 'preview', 'ready' and 'published').

Reply:
Though might be ok to ignore for the skeleton.